_posts/2017-11-23-omero-5-4-1.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of OMERO 5.4.1
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.1.
+---
+Today we are releasing OMERO 5.4.1, a bug-fix release.
+
+Improvements include:
+
+* labelled zoom slider bars in the UI to differentiate from horizontal
```

Comment:
FYI, in US English "labelled" *is acceptable* but unusual in comparison with "labeled".